contracts/access/README.adoc

```diff
@@ -5,7 +5,9 @@ NOTE: This document is better viewed at https://docs.openzeppelin.com/contracts/
 
 This directory provides ways to restrict who can access the functions of a contract or when they can do it.
 
-- {AccessControl} provides a general role based access control mechanism. Multiple hierarchical roles can be created and assigned each to multiple accounts.
+- {AccessManager} is a full-fledged role based access control solution smart contract systems. Allows creating and assigning multiple hierarchical roles with execution delays for each account across various contracts.
```

Comment:
> {AccessManager} is a full-fledged role based access control solution smart contract systems.

{AccessManager} is a [...] system. (singular) We should drop the "s". Also, the [...] part is unecessarily complex in my opinion. This should probably be rewitten into multiple sentences.

Reply:
Maybe something like:
{AccessManager} is a full-fledged role-based access-control solution.

(which is already a lot)

Reply:
Yeah I agree this sounds unnecessarily complex but my intention was:

```suggestion
- {AccessManager} is a full-fledged role based access control solution for smart contract systems. Allows creating and assigning multiple hierarchical roles with execution delays for each account across various contracts.
```

If it's still too much we can do:

```suggestion
- {AccessManager} is a full-fledged access control solution for smart contract systems. Allows creating and assigning multiple hierarchical roles with execution delays for each account across various contracts.
```

Also, note I tried to match this description with the description of the "AccessControl" item.